### DISCUSSION TITLE
Mapping between a XAML namespace and a CLR namespace in WinUI 3

### DISCUSSION THREAD

**Discussion by vladimir-stoyanov** [microsoft/WindowsAppSDK#2946] [Q&A]
Is it possible to map between a XAML namespace and a CLR namespace in WinUI 3? 

For example in WPF, the following attribute: [XmlnsDefinition](https://docs.microsoft.com/en-us/dotnet/api/system.windows.markup.xmlnsdefinitionattribute?view=windowsdesktop-6.0) is placed in the AssemblyInfo.cs file in order to achieve this. 

Is this achievable in WinUI 3 and if so how?

**Comment by peter0302** [comment]
Would love to know. It's there in `Microsoft.UI.Xaml.Markup` but useless as far as I can tell.
